default/python/AI/ResearchAI.py

```diff
@@ -701,16 +701,17 @@ def generate_classic_research_orders():
                 print "    Error: failed attempt to enqueued Tech: " + name
                 print "    Error: exception triggered and caught: ", traceback.format_exc()
 
-        print '\n\nAll techs:'
-        print '=' * 20
-        alltechs = fo.techs()
-        print_in_columns(sorted(fo.techs()), columns=3)
-
-        print '\n\nAll unqueued techs:'
-        print '=' * 20
-        # coveredTechs = new_tech+completed_techs
-        print_in_columns([tn for tn in alltechs if tn not in tech_base], columns=3)
-        print
+        if False:
```

Comment:
I think generally it is not very useful to spam logs with all the techs available in the game. But it may be useful for some specific testing so I don't want to remove the lines completely. I don't like to comment debug code out.

Would you prefer some switch ```print_all_techs=False``` to be used instead?